src/FluxCalculator.jl

```diff
@@ -324,20 +277,22 @@ function compute_surface_fluxes!(
     Interfacer.get_field!(csf.scalar_temp4, sim, Val(:beta))
     beta = csf.scalar_temp4
 
-    scheme_properties =
-        (; z0b = z0b, z0m = z0m, Ch = FT(0), Cd = FT(0), beta = beta, gustiness = FT(1))
-
-    input_args = (;
-        thermo_state_sfc,
-        thermo_state_atmos,
-        uₕ_int,
-        csf.z_int,
-        csf.z_sfc,
-        scheme_properties,
-        boundary_space,
-        surface_params,
+    # Set some scalars that we hardcode for now
+    gustiness = FT(1)
+
+    # Construct the SurfaceFluxes.jl container of inputs
+    inputs = @. SF.ValuesOnly(
```

Comment:
Thanks for modifying this. The result of this PR is consistent with the expected outcome when running in `CPUMultiThreaded`; will test and report the GPU result shortly.